app/views/ansible_repository/_repository_form.html.haml

```diff
@@ -47,7 +47,7 @@
         %span.help-block{"ng-show" => "angularForm.scm_url.$error.required"}
           = _("Required")
         %span.help-block{"ng-show" => "angularForm.scm_url.$error.urlValidation"}
-          = _("URL must include a protocol (http:// or https://) or be a valid SSH path (user@server:path or ssh://user@address:port/path)")
+          = _("URL must include a protocol (http://, https:// or file://) or be a valid SSH path (user@server:path or ssh://user@address:port/path)")
```

Comment:
I just realized this, but by changing this, do I have to update the translations that this line is associated with?

If so, I can always just drop this change if that is more trouble than it is worth.

Reply:
Translations are updated automaticly. So need for you to worry about. Translators will have "retranslate" it. But we do it even for typo fixes so your change is worth it :)

Reply:
Okay, cool.  Thanks for the info.

Was mostly unsure about the workflow for this kind of thing (as what is above is basically used as the translation key), and if having the translation change was going to require holding up the bugfix, I would have rather split those two work items up to allow fixing some text be a follow up.

But if not, then I am completely fine leaving this in.  Thanks again for the reponse!